docs/source/commandline/commandline-all.rst

```diff
@@ -928,95 +1148,176 @@ Core Video Options
+
+      |	The default on Windows is ``d3d``.
+      |
+      |	The default for Mac OS X is ``opengl`` because OS X is guaranteed to have a compliant OpenGL stack.
+      |
+      |	The default on all other systems is ``soft``.
+      |
 
 .. _mame-commandline-numscreens:
 
 **-numscreens** *<count>*
 
-	Tells MAME how many output windows to create. For most systems, a single output window is all you need, but some systems originally used multiple screens (*e.g. Darius and PlayChoice-10 arcade machines*). Each screen (up to 4) has its own independent settings for physical monitor, aspect ratio, resolution, and view, which can be set using the options below. The default is ``1``.
+    Tells MAME how many output windows to create.  For most systems, a single
+    output window is all you need, but some systems originally used multipl
+    screens (*e.g. Darius and PlayChoice-10 arcade machines*).  Each screen
+    (up to 4) has its own independent settings for physical monitor, aspect
+    ratio, resolution, and view, which can be set using the options below.
+
+    The default is ``1``.
 
 .. _mame-commandline-window:
 
 **-[no]window** / **-[no]w**
 
-	Run MAME in either a window or full screen. The default is OFF (**-nowindow**).
+    Run MAME in either a window or full screen.
+
+    The default is OFF (**-nowindow**).
 
 .. _mame-commandline-maximize:
 
 **-[no]maximize** / **-[no]max**
 
-	Controls initial window size in windowed mode. If it is set on, the window will initially be set to the maximum supported size when you start MAME. If it is turned off, the window will start out at the smallest supported size. This option only has an effect when the -window option is used. The default is ON (**-maximize**).
+    Controls initial window size in windowed mode.  If it is set on, the window
+    will initially be set to the maximum supported size when you start MAME.  If
+    it is turned off, the window will start out at the smallest supported size.
+    This option only has an effect when the -window option is used.
+
+    The default is ON (**-maximize**).
 
 .. _mame-commandline-keepaspect:
 
 **-[no]keepaspect** / **-[no]ka**
 
-	Enables aspect ratio enforcement. When this option is on, the system's proper aspect ratio (generally 4:3 or 3:4) is enforced, so you get the system looking like it should. When running in a window with this option on, you can only resize the window to the proper aspect ratio, unless you are holding down the CONTROL key. By turning the option off, the aspect ratio is allowed to float. In full screen mode, this means that all systems will stretch to the full screen size (even vertical systems). In window mode, it means that you can freely resize the window without any constraints. The default is ON (**-keepaspect**).
+    Enables aspect ratio enforcement. When this option is on, the system's
+    proper aspect ratio (generally 4:3 or 3:4) is enforced, so you get the
+    system looking like it should.
```

Comment:
This needs to be reworded.  Something along the lines of, "When enabled, MAME preserves the correct aspect ratio for emulated screen images..."  The part in parentheses should be "(usually 4:3 or 3:4 for CRT monitors)" because we have LCDs with all different aspect ratios, and we're starting to emulate workstations that used Sony 5:4 Trinitron CRTs.

If you don't do this one, I'll get to it on my next pass over this file.

Reply:
Corrected, though we can discuss phrasing if it's not to your liking.